### ISSUE TITLE
[Feature Request] Option to Show "Linked" and "Related" sidebar

### ISSUE THREAD

**Issue by ryanbuening** [elrumo/stackOverflow_focus#8]
Great extension! I was just wondering if anyone else would find it useful to have the option to show the "Linked" and "Related" sidebar sections. There are many times where I have found my solution from following the links in those sections.

![image](https://user-images.githubusercontent.com/812974/106912537-1ca97e00-66d1-11eb-8bb5-ab79db195d0a.png)

**Comment by elrumo** [comment]
Good idea, I'll see if I can add an option to keep it visible when I get some time :)
